src/shared/mock/index.ts

```diff
@@ -17,8 +17,16 @@ import {
   ParticipantWithDataAccess,
   SubjectDetailsWithDataAccess,
 } from 'modules/Dashboard/types';
-import { AssessmentActivityItem } from 'modules/Dashboard/features/RespondentData/RespondentDataReview';
+import { page } from 'resources';
 
+import {
+  defaultMultiSelectionConfig,
+  defaultSingleSelectionConfig,
+  defaultSliderConfig,
+  defaultTextConfig,
+  defaultTimeConfig,
+} from '../../modules/Builder/features/ActivityItems/ItemConfiguration/OptionalItemsAndSettings/OptionalItemsAndSettings.const';
+import { LookupTableDataItem } from '../../modules/Builder/features/ActivitySettings/SubscalesConfiguration/LookupTable';
```

Comment:
Nit: sometimes relative imports sneak in where they're not supposed to, it's hard to get VS Code to behave consistently!

```suggestion
} from 'modules/Builder/features/ActivityItems/ItemConfiguration/OptionalItemsAndSettings/OptionalItemsAndSettings.const';
import { LookupTableDataItem } from 'modules/Builder/features/ActivitySettings/SubscalesConfiguration/LookupTable';
```

Reply:
Updating 🤓